test/unit-tests/test-output/_utils.ts

```diff
@@ -0,0 +1,5 @@
+export function getDummyStream(): any {
```

Comment:
Happy to move it across if you'd prefer (it's only changing the import and moving it) but I think it'd be best to keep it as a "utility function" rather than a builder until it's fully fleshed out.